pkgs/development/python-modules/denonavr/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests, pytest, testtools, requests-mock }:
+
+buildPythonPackage rec {
+  pname = "denonavr";
+  version = "0.7.10";
```

Comment:
```suggestion
  version = "0.7.10";
  disabled = isPy27; # no module html
```

```
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
builder for '/nix/store/6h2fnr4d6blkdz7jwhqxp9r53fzplww9-python2.7-denonavr-0.7.10.drv' failed with exit code 2; last 10 log lines:
  Traceback:
  tests/test_denonavr.py:13: in <module>
      import denonavr
  /nix/store/ky4q7k4k7by0bk2p8ynlraza0mdfqm5c-python2.7-denonavr-0.7.10/lib/python2.7/site-packages/denonavr/__init__.py:14: in <module>
      from .denonavr import DenonAVR
  /nix/store/ky4q7k4k7by0bk2p8ynlraza0mdfqm5c-python2.7-denonavr-0.7.10/lib/python2.7/site-packages/denonavr/denonavr.py:15: in <module>
      import html
  E   ImportError: No module named html
  !!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
  =========================== 1 error in 0.25 seconds ============================
cannot build derivation '/nix/store/nnzlfdqpc0c4vp8gbg2l3ggq92zh1dpz-env.drv': 1 dependencies couldn't be built
[3 built (1 failed), 0.0 MiB DL]
error: build of '/nix/store/nnzlfdqpc0c4vp8gbg2l3ggq92zh1dpz-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/78012
1 package failed to build:
python27Packages.denonavr

2 package built:
python37Packages.denonavr python38Packages.denonavr
```